packages/core-data/src/fetch/index.js

```diff
@@ -8,7 +8,7 @@ import { camelCase } from 'change-case';
  */
 import apiFetch from '@wordpress/api-fetch';
 
-export { default as __experimentalFetchLinkSuggestions } from './__experimental-fetch-link-suggestions';
+export { default as fetchLinkSuggestions } from './fetch-link-suggestions';
```

Comment:
Do we really want to make this public?

Reply:
It's been public since WP 5.8, so I'm removing `__experimental` to reflect that it's stable.

Reply:
I don't think I agree that, because something has been experimental for a long time, it should be stable.

Reply:
I think we should strive to eventually stabilise, deprecate, or privatise all of our `__experimental` APIs. This one has existed for 3 years without much change, could be of some use to a third party, and is available in core, so I think stabilise is the best of those options.

To flip the question: why _not_ stabilise this?

Not essential at all to this PR, just trying to tidy the codebase incrementally 🙂

Reply:
I think it's fine to stabilize for all the reasons @noisysocks mentions. It's already public and is used in core.

Reply:
Where? https://github.com/search?q=repo%3AWordPress%2Fwordpress-develop%20__experimentalFetchLinkSuggestions&type=code

Reply:
As part of Gutenberg.

Reply:
You can check but disabling Gutenberg and running `wp.coreData.__experimentalFetchLinkSuggestions` in DevTools.

Reply:
Yes, I know it's there, but @ntsekouras said _used_ in core, which I can't find. Yes it is used internally in GB, for which we have private APIs.

Reply:
Right. No, core doesn't use it, but it's available for plugins and sites to use.

Do you think `fetchLinkSuggestions` should be a private API?

To me it seems like a useful API for third parties that want to hook up `@wordpress/block-editor` in a WordPress environment, so even if we discount the "it already has been public for some time" argument, I'd lean towards stabilising it.

Reply:
Ended up deciding to keep this experimental for now anyway. https://github.com/WordPress/gutenberg/pull/62397#issuecomment-2167265679